components/WeeklyPickButton/WeeklyPickButton.tsx

```diff
@@ -7,6 +7,7 @@ import { Label } from '../Label/Label';
 import { RadioGroupItem } from '../RadioGroup/RadioGroup';
 
 type WeeklyPickButtonProps = {
+  homeAway: string;
```

Comment:
nit: I feel like this is not a very clear prop name, and, while this is a p1 fix, maybe we should revisit it in the future.

Reply:
its already being prop drilled from another component as "competition.homeAway" and homeAway is a prop from an interface [] with 20 other props lol. but the actual prop itself is just a string.

Reply:
So obviously leave here as is. I wonder if this is worth putting as a ticket. @shashilo, thoughts?

Reply:
I'm fine with this.